packages/web/package.json

```diff
@@ -67,6 +67,7 @@
     "@types/lodash.memoize": "^4.1.7",
     "@types/react-helmet": "^6.1.5",
     "clsx": "^1.1.1",
+    "cross-fetch": "^3.1.5",
```

Comment:
There's already a `node-fetch` package in `shared`. I think we better use same packages across the monorepo.

Reply:
`cross-fetch` is a maintained version of `isomorpgic-fetch`. They both use `node-fetch` internally, turning it into a polyfill. I'm rather unsure if we want to fiddle with globals ourselves instead of relying on a package.

Reply:
Ok, then may I ask you to replace node-fetch with cross-fetch in `packages/web/src/services/apiClient.ts` too?

Reply:
Since it's a proper server polyfill, I'm dropping explicit references to `node-fetch` altogether.